server/controllers/users/password.js

```diff
@@ -29,9 +29,10 @@ export const forgot = async function(req, res) {
   if (!user) {
     throw new ValidationError({email: 'No account with that email has been found'})
   } else if (user.provider !== 'local') {
-    return res.status(400).send({
+    if (user.provider === 'google') await mailer.sendPasswordGoogle(user)
+    /*return res.status(400).send({
       message: 'It seems like you signed up using your ' + user.provider + ' account'
-    })
+    })*/
   } else {
     user.resetPasswordToken = token
     user.resetPasswordExpires = Date.now() + 3600000 // 1 hour
```

Comment:
and the mailer.sendPasswordReset has to be in this else block, or it'll be called for google users too